analysis/gradle.properties

```diff
@@ -9,3 +9,11 @@ org.gradle.parallel=true
 # Only relevant projects are configured which results in faster builds for large multi-projects.
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:configuration_on_demand
 org.gradle.configureondemand=true
+
+systemProp.sonar.organization=maibornwolff-gmbh
+systemProp.sonar.projectKey=maibornwolff-gmbh_codecharta_analysis
+systemProp.sonar.projectName=CodeCharta Analysis
+systemProp.sonar.host.url=https://sonarcloud.io
+systemProp.sonar.links.ci=https://travis-ci.org/MaibornWolff/codecharta
```

Comment:
This link is outdated

Reply:
Good catch, I copied those settting from the original config. Will be removed next push.